Updated BLAST/Nomad Books/Creeper Spores/Illuminations Wiki Website (#18)
Ladysnake/Ladysnake.github.io: wiki/illuminations.md

# Illuminations

**Illuminations** is a client sided Minecraft Fabric mod that adds multiple ambient particles in order to make your world more enjoyable at night or in dark places.
This mod is compatible with the [Canvas Renderer](https://www.curseforge.com/minecraft/mc-mods/canvas-renderer).

> *(All screenshots with a bloom effect were taken with Canvas Renderer)*

Reviewer: ```suggestion
*(All screenshots with a bloom effect were taken with [Canvas Renderer](https://www.curseforge.com/minecraft/mc-mods/canvas-renderer) and [Lumi Lights](https://spiralhalo.github.io/)*
```

> *(All screenshots with a bloom effect were taken with [Canvas Renderer](https://www.curseforge.com/minecraft/mc-mods/canvas-renderer))*

Reviewer: ```suggestion
*(All screenshots with a bloom effect were taken with Canvas installed)*
```
ok I'm stupid you linked it right above

> *(All screenshots with a bloom effect were taken with [Canvas Renderer](https://www.curseforge.com/minecraft/mc-mods/canvas-renderer) and [Lumi Lights](https://spiralhalo.github.io/)*

Reviewer: ```suggestion
*(All screenshots with a bloom effect were taken with [Canvas Renderer](https://www.curseforge.com/minecraft/mc-mods/canvas-renderer))*
```
